app/services/idv/phone_confirmation_session.rb

```diff
@@ -62,6 +73,7 @@ def to_h
         phone: phone,
         sent_at: sent_at.to_i,
         delivery_method: delivery_method,
+        user: user&.id,
```

Comment:
I would serialize this as `user_id` to make it clearer it's not the whole user
```suggestion
        user_id: user&.id,
```

Reply:
yup, done in f4541b2